amundsen_application/static/.eslintcache

```diff
@@ -0,0 +1 @@
+[{"/Users/miglesiasvalle/src/amundsenfrontend-private/upstream/amundsen_application/static/jest.config.js":"1"},{"size":974,"mtime":1591648575193,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"4aqx17","/Users/miglesiasvalle/src/amundsenfrontend-private/upstream/amundsen_application/static/jest.config.js",[]]
```

Comment:
Is this hanging around in your local environment from your linting PR that needs to be git ignored?

Reply:
Yeah, will do!